docs/ARCHITECTURE.md

```diff
@@ -50,6 +50,28 @@ service manager.
 
 BPF helpers written in C and used by PID 1 can be found under `src/core/bpf/`.
 
+### Implementing Unit Settings
+
+The system and session manager supports a large number of settings. These can generally
+be configured in three ways:
+
+1. Via textual, INI-style configuration files called `unit files`
```

Comment:
markup for unit files should probably be `*unit* *files`, i.e. not a teletype font, but an emphasis

Reply:
done